### DISCUSSION TITLE
Can I contribute to the documents?

### DISCUSSION THREAD

**Discussion by Foo-x** [reearth/reearth-visualizer#86] [Q&A] [answered]
The documentation pages are not responsive, and I found some typos on them.
https://docs.reearth.io/developer-guide on iPhone X
![docs reearth io_developer-guide(iPhone X)](https://user-images.githubusercontent.com/24248467/127467070-79635c76-25b8-4fc3-9603-1c0326d04edd.png)

I would like to contribute, but there doesn't seem to be a repo.
Do you have a plan to manage the documentation on GitHub?

**Comment by rot1024** [comment, marked as answer]
Thank you for posting.
We already started working on fixing that issue.

Now the repository is private, but we have a plan to make the repository public soon.

**Reply by rot1024** [reply]
Hello, Now documentation page has been renewed! https://docs.reearth.io

Also documentation repository is now open: https://github.com/reearth/docs.reearth.io

Feel free to contribute to docs. Thank you!
